### DISCUSSION TITLE
Crazy but everything seems to work and is running... but no idea if that is true?

### DISCUSSION THREAD

**Discussion by thadguidry** [DNSCrypt/dnscrypt-proxy#2319] [Q&A] [answered]
Running dnscrypt-proxy on Windows 11 and I thoroughly read the getting started guide, followed it to the letter, and reviewed completely (but did not change) the configuration TOML file.
And seemingly everything *seems* to be working?
But I honestly don't know how to actually verify this.
One thing that puzzles me is I'm not sure if my Network Settings for Ethernet adapter are correct?
![image](https://user-images.githubusercontent.com/986438/218288403-1e50d649-cfb0-470d-80f0-6a8c4861c140.png)

Prior to running dnscrypt-proxy, I had problems with reaching lots of sites such as ex. `raw.githubusercontent.com` because of GFW DNS poisoning but since running dnscrypt-proxy, things are just magically working well?
Truely, how can I be sure I setup everything well enough?   Thanks in advance, not a DNS expert.

**Comment by lifenjoiner** [comment, marked as answer]
Seems you set it right, although I'm not on Win11 exactly.

> But I honestly don't know how to actually verify this.

Is this what you want?
https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Checking

If you really want to capture the querying traffic, you can use WireShark on Windows.

**Reply by thadguidry** [reply]
Yes, that's a great wiki article. Thanks for pointing me to it! Indeed, it seems I didn't see or notice the Wiki page outline on the right hand side of the Github wiki pages.
